dallinger/recruiters.py

```diff
@@ -145,6 +154,26 @@ def recruit_participants(self, n=1):
             duration_hours=self.config.get('duration')
         )
 
+    def notify_recruited(self, participant, experiment):
+        """Assign a Qualification to the Participant for the experiment ID,
+        and for the configured group_name, if it's been set.
+        """
+        score = '1'  # always '1'; doesn't matter
```

Comment:
In the original issue Monica mentioned the possibility of allowing participants to participate multiple times but with a finite upper limit. Using the value/score of the qualification to track this seems the easiest way to do this. So when participants first arrive, and they dont have the relevant qualification, we assign them the qualification with a value of 1. When they take part again, we update the value of their qualification, incrementing by 1 each time. Once they reach the threshold they are prevented from taking part again.

A nice outcome is that by setting the threshold to 1 we get the standard behavior and we can set this as the default value unless the experimenter overwrites it.

An unpleasant outcome is that I am not sure you can tell MTurk that participants need a qualification with a value less than a certain threshold to view/accept the hit so we might have to handle this on the ad page ourselves

@suchow thoughts?